tests/10_Writing/CopyMethodsTest.php

```diff
@@ -186,16 +186,42 @@ public function testCopyNoUpdateOnCopy()
         $this->ws->copy($src, $dst);
     }
 
-    public function testCopyUpdateOnCopy()
+    /**
+     * Copied nodes which reference other nodes should be shown in the referrers list of references
+     * Single value
+     */
+    public function testCopyUpdateReferencesSingleValue()
     {
-        $src = '/tests_write_manipulation_copy/testCopyUpdateOnCopy/srcNode';
-        $dst = '/tests_write_manipulation_copy/testCopyUpdateOnCopy/dstNode/srcNode';
+        $src = '/tests_write_manipulation_copy/testCopyUpdateReferrersSingleValue/srcNode';
+        $dst = '/tests_write_manipulation_copy/testCopyUpdateReferrersSingleValue/dstNode';
+        $ref = '/tests_write_manipulation_copy/testCopyUpdateReferrersSingleValue/referencedNode';
+
         $this->ws->copy($src, $dst);
```

Comment:
we should have the assertions before and after refreshing, to make sure both the object state and the storage state are correctly updated.